src/styles/globals.scss

```diff
@@ -227,11 +213,20 @@
     --Nav-navLink-border-color: var(--sds-clr-lilac-09);
     pointer-events: none;
   }
+
+  &__icon {
+    display: flex;
+
+    // to fix Safari only styling bug
+    width: pxToRem(14px);
+    height: pxToRem(14px);
+  }
 }
 
 .SidebarLink {
   --SidebarLink-color: var(--sds-clr-gray-11);
 
+  justify-content: space-between;
```

Comment:
We don't want to apply this style to all links in the sidebar, but only the ones we want to expand, right?

![image](https://github.com/user-attachments/assets/d3d887ba-9374-41b7-b5bd-6f506e45e1cb)